modules/packer/custom-image/variables.pkr.hcl

```diff
@@ -98,13 +98,18 @@ variable "service_account_email" {
   default     = null
 }
 
-variable "service_account_scopes" {
+variable "scopes" {
```

Comment:
This is a breaking change. I assume you changed the name so that it would match the google compute. I earnest I don't think we can make this change without rolling version number to 2.0 on our next release. My suggestion would be to leave the name what it was, even though there is a mismatch. Mismatch aside it is still a reasonable name for the variable. 

If you think it is critical to have symmetry with the `googlecompute` API then let's talk change management offline.

Reply:
Although I realize you are focusing on the variable name, I think one could reasonably describing the set of choices as a "fixing" change since the status quo is to do nothing with the supplied values. If there were a good Packer equivalent to `tflint` this would have been caught during its introduction. We can discuss further offline to handle this.

Reply:
I agree with you in this specific case as it is impossible that users could be negatively impacted by this change as it was never working to start.